.github/workflows/full-ci-cd.yml

```diff
@@ -23,3 +23,42 @@ jobs:
     needs: run_e2e_tests
     uses: NYPL/sfr-bookfinder-front-end/.github/workflows/build-production.yaml@main
     secrets: inherit
+
+  # Rollback job in case of failure (Revert production to the previous task definition)
+  rollback:
```

Comment:
What are your thoughts on creating a separate workflow for the rollback? 

I think you'll need to add an on block: 

```
on:
  workflow_dispatch:
...
```